### DISCUSSION TITLE
Will there be support for ARMv7?

### DISCUSSION THREAD

**Discussion by NuclearGeneral** [hassio-addons/addon-vscode#200] [Q&A]
Hello,

Would there happen to be plans to implement this for support of ARMv7 cpu architecture?

Thanks !

**Comment by frenck** [comment]
Nope. The devices Home Assistant supports and would run this, barely can handle this add-on. I'm not willing to support the issues flowing out of that. Furthermore, 64 bits is recommended by Home Assistant nowadays.
